docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,164 @@
+# Enabling the OpenStack CLI tool
+
+The OpenStack CLI tool, also known as OpenStack Command Line Client, 
+conveniently provides access to various OpenStack APIs. Using the 
```

Comment:
I'm always a big fan of expanding acronyms and initialisms on first mention. Thus, I'd put "The OpenStack command line interface (CLI), also known as..."

At that point however the re-mention of command line becomes an ugly redundancy, so let's go with

```markdown
The OpenStack command line interface (CLI), also known as OpenStack Client (OSC) or
simply `openstack`, conveniently provides access to various OpenStack APIs. Using the
CLI, you remotely create and manage [...]
```

Reply:
Aha, yes, that's the way to go!